server/routes/creatingLicences/handlers/checkAnswers.ts

```diff
@@ -30,8 +32,23 @@ export default class CheckAnswersRoutes {
     const { conditionsWithUploads, additionalConditions } = this.conditionService.additionalConditionsCollection(
       licence.additionalLicenceConditions
     )
+    parentOrSelfAdditionalAPConditions = additionalConditions
 
-    res.render('pages/create/checkAnswers', { additionalConditions, conditionsWithUploads, backLink })
+    const inPssPeriod = isInPssPeriod(licence)
+
+    if (inPssPeriod) {
+      const { additionalConditions: parentAdditionalConditions } = this.conditionService.additionalConditionsCollection(
+        (await this.licenceService.getParentLicenceOrSelf(licence.id, user))?.additionalLicenceConditions
+      )
+      parentOrSelfAdditionalAPConditions = parentAdditionalConditions
+    }
+
+    res.render('pages/create/checkAnswers', {
+      parentOrSelfAdditionalAPConditions,
+      conditionsWithUploads,
```

Comment:
I _think_ we need to get the `conditionsWithUploads` from the parent licence too for the `inPssPeriod` situation, in the same way you have with `additionalConditions` on line 35.
In other words, on line 40 it'll need to be `{ conditionsWithUploads, additionalConditions }` (the same as line 32), and you'll need to create a `parentOrSelfConditionsWithUploads` on line 44, which you pass in instead of `conditionsWithUploads` on line 48.

Alternatively rather than creating the `parentOrSelf-` versions of the vars, line 32 could be changed to a `let` and simply reassign them in the `inPssPeriod` condition if necessary.
Not sure if `let` is problematic to use in this case or not, @andrewrlee what do you think?

Reply:
ok. If we need conditionsWithUploads from the parent then we can completely refactor it.
I removed the parentOrSelf- variable and i am reassigning to the same destructured object.

Reply:
I've just realised I didn't actually explain why
Basically, one of the conditions (the one about exclusion zones were you upload a map of an area that the offender cannot enter) works a bit differently to the others. The requirement was to add multiple exclusion zones (this is what people mean if you hear the acronym 'MEZ') to one condition, but the with the templating system that we use this proved to be very difficult. The solution that was found was to duplicate the licence condition for each map (so an offender can have multiple entries of the same condition), then when rendering, combine all of the instances of the MEZ condition into one (this probably makes more sense if you see an example of how it looks!)

Currently, only the exclusion zone condition has uploads, so whenever you see something like `conditionsWithUploads`, it basically means `instances of the exclusion zone condition`. This means that by not including it here, the result would be that the exclusion zone condition would never appear in the new 'expired conditions' section that you've added, which is very easy to miss in testing if you happen to just not select that condition during your tests!